docusaurus/docs/get-started/get-started.mdx

```diff
@@ -135,82 +167,48 @@ Depending on the answers you gave to the prompts, there should now be a structur
 └── tsconfig.json
 ```
 
-When you've finished installing the tools, open the plugin folder:
-
-<CodeSnippets
-  snippets={[
-    { component: InstallNPM, label: 'npm' },
-    { component: InstallPNPM, label: 'pnpm' },
-    { component: InstallYarn, label: 'yarn' },
-  ]}
-  groupId="package-manager"
-  queryString="current-package-manager"
-/>
-
-## Create-plugin prompts reference
-
-When running the `create-plugin` command, the following prompts appear:
+For more information about these files, refer to [Folder structure](get-started/folder-structure/).
 
-### What is the name of your plugin?
+### Step 3: Run your plugin in Docker
 
-Enter the name of your plugin. This helps to identify its purpose.
+With the `create-plugin` tool, you can use a Docker container to simplify the configuration, loading, and development processes. For more information, refer to [Set up development environment](get-started/set-up-development-environment/).
 
-### What is the organization name of your plugin?
+### Step 4: Use CLI commands
 
-Enter the name of your organization. Grafana plugins require an organization name (normally your [Grafana account](https://grafana.com/signup/) username) to help uniquely identify your plugin.
+After you have installed `create-plugin`, you can run built-in commands in the shell. Refer to the section below.
 
-### How would you describe your plugin?
+## Frequently used CLI commands
 
-Give your plugin a description. This will help users more easily understand what it does when the plugin is distributed.
+Use the CLI for essential tasks of plugin development.
```

Comment:
I would add a note here to tell that these are `npm` or `yarn` commands (so the full comand is for example `npm run build`)

Reply:
done.